Sources/OpenCombineFoundation/Publishers+Timer.swift

```diff
@@ -0,0 +1,345 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2019 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+// Only support 64bit
+#if !(os(iOS) && (arch(i386) || arch(arm)))
+
+import Foundation
+import OpenCombine
+
+#if canImport(COpenCombineHelpers)
+import COpenCombineHelpers
+#endif
+
+private typealias Lock = __UnfairLock
```

Comment:
We already have the needed typealias here:
https://github.com/broadwaylamb/OpenCombine/blob/d680f09932fe68942c3d391d6df296d38b9dcd05/Sources/OpenCombineFoundation/Helpers/Locking.swift#L14-L15